Jenkinsfile

```diff
@@ -1,24 +1,53 @@
 elifePipeline {
     node('containers-jenkins-plugin') {
         def commit
+        def grobidTag
+        def fullImageTag
 
         stage 'Checkout', {
             checkout scm
             commit = elifeGitRevision()
+            grobidTag = sh(
+                script: 'bash -c "source .env && echo \\$GROBID_TAG"',
+                returnStdout: true
+            ).trim()
+            echo "GROBID_TAG: ${grobidTag}"
+            assert grobidTag != ''
+            fullImageTag = "${grobidTag}-${commit}"
+            echo "Full image tag: ${fullImageTag}"
         }
 
         stage 'Build and run tests', {
             try {
-                sh "make ci-build-and-test"
+                sh "IMAGE_TAG=${fullImageTag} make ci-build-and-test"
             } finally {
-                sh "make ci-clean"
+                sh "IMAGE_TAG=${fullImageTag} make ci-clean"
             }
         }
 
+        stage 'Check GROBID label', {
+            echo "Checking GROBID label..."
+            def image = DockerImage.elifesciences(this, 'sciencebeam-trainer-grobid', fullImageTag)
+            echo "Reading GROBID label of image: ${image.toString()}"
```

Comment:
thought that `.toString()` was automatic when used in string interpolation, but must be another programming language